src/gui/gui.h

```diff
@@ -1611,6 +1611,7 @@ class FurnaceGUI {
 
   // wave generator
   int waveGenBaseShape;
+  int waveInterpolation;
```

Comment:
Please add an initializer for this in gui.cpp. Search for `waveGenBaseShape(` and put the initializer next to it.

Reply:
I added it, thanks for this advice.
I just tested the feature with this change, it works fine.